Userland/Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -551,9 +552,12 @@ JS_DEFINE_NATIVE_FUNCTION(ArrayPrototype::concat)
                     return;
                 if (k_exists) {
                     auto k_value = obj.get(k).value_or(js_undefined());
-                    if (vm.exception())
+                    if (vm.exception()) {
                         return;
+                    }
```

Comment:
Please don't add curly braces to a single-line if body.

Reply:
Whoops left over from debugging.
Removed them